components/template/vue/vue-template.ts

```diff
@@ -4,7 +4,7 @@ import {
 } from '@bit/bit.javascript.default-generator.canonize';
 import { CodeGenerator } from '@bit/bit.javascript.raw.code-generator';
 
-export const mainFile = 'index.vue';
+export const mainFile = 'Index.vue';
```

Comment:
Please leave lowercase, as the Vue Capsule imports `./src/index.vue` and may break.

Plus, mixed casing may cause troubles in Windows, so we always prefer lower case `kebab-case.tsx` for files

Reply:
done

Reply:
nice